src/GitVersionCore.Tests/AssemblyInfoFileUpdaterTests.cs

```diff
@@ -4,482 +4,486 @@
 using System.Linq;
 using GitVersion;
 using GitVersion.Helpers;
-using GitVersionCore.Tests;
 using NSubstitute;
 using NUnit.Framework;
 using Shouldly;
+using GitVersionCore.Tests;
 
-[TestFixture]
-public class AssemblyInfoFileUpdaterTests
+namespace GitVersionExe.Tests
```

Comment:
The changes here seem to indicate the forward integration master missed some things, particularly around the changes I made in #1249.

Reply:
The fact that the approve files above all are showing changes seems to be related to this as well.

Reply:
> The changes here seem to indicate the forward integration master missed some things, particularly around the changes I made in #1249.

Yeah.. Looks like I had a merge conflict which I resolved in Commit 42d253e7 by adding a namespace and some other minor changes that you see. GitHub shows massive changes to that file, but a VS compare shows only serveral lines were changed. However I have no idea why I made those changes so perhaps it was unintentional. I have just reverted the file to match master version anyhow.

> The fact that the approve files above all are showing changes seems to be related to this as well.

They are all listed as "No changes." though. Perhaps it shows them just because they were renamed / relocated in master.